NavigateU/NavigateU/Vendor/mappings/questions.json

```diff
@@ -0,0 +1,11 @@
+
+{
+  "request": {
+      "method": "GET",
+      "url": "/questions"
+  },
+  "response": {
+      "status": 200,
+      "bodyFileName": "questions.json"
+  }
+}
```

Comment:
Can you explain please in telegram why do you need this files? As i remember, you don't need them to interact with MockApi